components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -81,27 +84,54 @@ export default {
 	},
 };
 
+// ********* DOCS ONLY ********* //
+
 export const Default = ActionButtonGroup.bind({});
-Default.args = {
-	label: "More",
+Default.args = {};
+Default.tags = ["!autodocs"];
+
+/**
+ * Action buttons should always have a label, unless they are only using an icon that is universally understood and accessible. They can have an optional icon, but it should not be used for decoration. Use an icon only when necessary and when it has a strong association with the label text.
+ *
+ * The label can be hidden to create an icon-only action button. If the label is hidden, an icon is required, and the label will appear in a tooltip on hover.
+ */
+
+export const Standard = TreatmentTemplate.bind({});
+Standard.args = Default.args;
+Standard.tags = ["!dev"];
+Standard.parameters = {
+	chromatic: { disableSnapshot: true },
 };
 
-export const Emphasized = ActionButtons.bind({});
+Standard.storyName = "Default";
+
+/**
+ * By default, action buttons are not emphasized. This is optimal for when an action button is not the core part of an interface, such as in application panels, where all the visual components are monochrome in order to direct focus to the content.
```

Comment:
@aramos-adobe wanna add anything about having [a quiet-emphasized action button,](https://github.com/adobe/spectrum-css/pull/3102#discussion_r1755276853) to cover that from the static docs site?

And similar to my comment on the quiet story, we could think about trimming some of this emphasized documentation down, focusing on the emphasized paragraph and not the default stuff, to clarify.